frontend/src/stories/components/PersonalSchedules/SinglePersonalScheduleDropdown.stories.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+import SinglePersonalScheduleDropdown from "main/components/PersonalSchedules/SinglePersonalScheduleDropdown";
+import { personalScheduleFixtures } from "fixtures/personalScheduleFixtures";
+// import { personalSchedulesFixtures } from "fixtures/personalSchedulesFixtures";
+
+
+export default {
+    title: 'components/PersonalSchedules/SinglePersonalScheduleDropdown',
+    component: SinglePersonalScheduleDropdown
+};
+
+const Template = (args) => {
+    const [personalSchedules, setPersonalSchedule] = useState(args.personalSchedules[0]);
+
+    return (
+        < SinglePersonalScheduleDropdown 
+        personalSchedules={personalSchedules} 
+        setPersonalSchedule={setPersonalSchedule} 
+        controlId={"SampleControlId"}
+        label={"Personal Schedule"} 
+        {...args} />
+    )
+};
+
+// export const OnePersonalSchedule = Template.bind({});
+// OnePersonalSchedule.args = {
+//     personalSchedules: personalSchedulesFixtures.onePersonalSchedule
+// };
```

Comment:
Commented out code should be removed before you submit a PR for review.

And code reviewers should have caught this and asked for it to be removed.